bootstrap/sql/migrations/native/1.5.0/mysql/postDataMigrationSQLScript.sql

```diff
@@ -24,6 +24,80 @@ WHERE name IN (
     'tableRowCountToBeBetween'
 );
 
+-- Update DeltaLake service due to connection schema changes to enable DeltaLake ingestion from Storage
+UPDATE dbservice_entity dbse
+SET
+  dbse.json = JSON_REMOVE(JSON_REMOVE(
+  JSON_MERGE_PATCH(
+    dbse.json,
+    JSON_OBJECT(
+      'connection', JSON_OBJECT(
+        'config', JSON_OBJECT(
+          'configSource', JSON_OBJECT(
+            'connection', JSON_EXTRACT(dbse.json, '$.connection.config.metastoreConnection'),
+            'appName', JSON_UNQUOTE(JSON_EXTRACT(dbse.json, '$.connection.config.appName'))
+          )
+        )
+      )
+    )
+  )
+  , '$.connection.config.appName'), '$.connection.config.metastoreConnection')
+WHERE dbse.serviceType = 'DeltaLake';
```

Comment:
@ulixius9 I think something happened when updating the branch since this shouldn't be here. It's about another PR and it was already moved: https://github.com/open-metadata/OpenMetadata/pull/16789/files

Reply:
Oh I see might have messed the merge